File: src/codes/Users_Manual/gain/holo_0.py
from pyautd3.gain.holo import GSPAT, NalgebraBackend, Pa

~import numpy as np
~x1 = 0.0
~y1 = 0.0
~z1 = 0.0
~x2 = 0.0
~y2 = 0.0
~z2 = 0.0
backend = NalgebraBackend()
g = GSPAT(backend, [(np.array([x1, y1, z1]), 5e3 * Pa), (np.array([x2, y2, z2]), 5e3 * Pa)])

File: src/codes/Users_Manual/gain/holo_0.cs
using AUTD3Sharp.Gain.Holo;

~using AUTD3Sharp.Utils;
~using static AUTD3Sharp.Units;
~var x1 = 0.0f;
~var y1 = 0.0f;
~var z1 = 0.0f;
~var x2 = 0.0f;
~var y2 = 0.0f;
~var z2 = 0.0f;
var backend = new NalgebraBackend();
var g = new GSPAT(backend, [
                        (new Vector3(x1, y1, z1), 5e3f * Pa),
                        (new Vector3(x1, y1, z1), 5e3f * Pa)
                    ]);